PaddleCV/object_detection/README.md

```diff
@@ -10,63 +13,61 @@ This object detection framework is based on PaddlePaddle. We want to provide cla
+- Performance Optimized:
+with the help of the underlying PaddlePaddle framework, faster training and
+reduced GPU memory footprint is achieved. Notably, Yolo V3 training is
+much faster compared to other frameworks. Another example is Mask-RCNN
+(ResNet50), which can be trained with batch size of 5 on a single Tesla V100
+(16G) GPU.
 
-The supported architectures are as follows:
+Supported Architectures:
 
-|                    | ResNet |ResNet vd| ResNeXt  | SENet    | MobileNet | DarkNet|
-|--------------------|:------:|--------:|:--------:|:--------:|:---------:|:------:|
-| Faster R-CNN       | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
-| Faster R-CNN + FPN | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
-| Mask R-CNN         | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
-| Mask R-CNN + FPN   | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
-| Cascade R-CNN      | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
-| RetinaNet          | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
-| Yolov3             | ✓      | ✗      | ✗         |  ✗       | ✓        | ✓     |
-| SSD                | ✗      | ✗      | ✗         |  ✗       | ✓        | ✗      |
+|                    | ResNet | ResNet-vd <sup>[1](#vd)</sup> | ResNeXt | SENet | MobileNet | DarkNet |
+|--------------------|:------:|------------------------------:|:-------:|:-----:|:---------:|:-------:|
+| Faster R-CNN       | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Faster R-CNN + FPN | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Mask R-CNN         | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Mask R-CNN + FPN   | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| Cascade R-CNN      | ✓      |                             ✓ | ✓       | ✓     | ✗         | ✗       |
+| RetinaNet          | ✓      |                             ✗ | ✗       | ✗     | ✗         | ✗       |
+| Yolov3             | ✓      |                             ✗ | ✗       | ✗     | ✓         | ✓       |
+| SSD                | ✗      |                             ✗ | ✗       | ✗     | ✓         | ✗       |
 
-The extensive capabilities:
+<a name="vd">[1]</a> ResNet-vd models offer much improved accuracy with negligible performance cost.
 
-- [x] **Synchronized batch norm**:  used in Yolo v3.
-- [x] **Group Norm**: supported this operation, the related model will be added later.
-- [x] **Modulated deformable convolution**: supported this operation, the related model will be added later.
-- [x] **Deformable PSRoI Pooling**: supported this operation, the related model will be added later.
+Advanced Features:
 
+- [x] **Synchronized batch norm**: currently used by Yolo V3.
+- [x] **Group Norm**: pretrained models to be released.
```

Comment:
Group Norm -> Group norm

Reply:
on the contrary, I think they should be capitalized...

Reply:
capitalized

Reply:
pretrained models to be released.  -> supported, pretrained models to be released. ?

Reply:
IMHO, putting them in the "advanced features" section implies that they are supported, we only need to clarify that the pretrained models are not available at the moment.